setup.cfg

```
@@ -101,9 +101,11 @@ install_requires =
     # Serious breaking changes. Flask 2.0 has been introduced in May 2021 and 2.0.2 version is available
     # now (Feb 2022): TODO: we should attempt to migrate to Flask 2 and all below flask libraries soon.
     flask>=1.1.0, <2.0
-    # FlaskAppBuilder is very tight integration for UI but we are likely to remove it as a dependency soon
-    # TODO: Remove it when we are ready
-    flask-appbuilder~=3.4, <4.0.0
+    # We are tightly coupled with FAB version because we vendored in part of FAB code related to security manager
+    # This is done as part of preparation to removing FAB as dependency, but we are not ready for it yet
+    # Every time we update FAB version here, please make sure that you review TODO @jhtimmins - please describe 
+    # what to compare and how to sync the changes :)
```

```suggestion
    # Every time we update FAB version here, please make sure that you review the classes and models in
    # `airflow/www/fab_security` with their upstream counterparts. In particular, make sure any breaking changes,
    # for example any new methods, are accounted for.
```

I took a stab at this and @jhtimmins can expand/correct it.